polkadot/transaction-pool/src/lib.rs

```diff
@@ -279,13 +279,16 @@ impl<'a, A> txpool::Verifier<UncheckedExtrinsic> for Verifier<'a, A> where
 	type Error = Error;
 
 	fn verify_transaction(&self, uxt: UncheckedExtrinsic) -> Result<Self::VerifiedTransaction> {
-		info!("Extrinsic Submitted: {:?}", uxt);
 
 		if !uxt.is_signed() {
 			bail!(ErrorKind::IsInherent(uxt))
 		}
 
-		let (encoded_size, hash) = uxt.using_encoded(|e| (e.len(), BlakeTwo256::hash(e)));
+		let encoded = uxt.encode();
+		let (encoded_size, hash) = (encoded.len(), BlakeTwo256::hash(&encoded));
+		
+		debug!("Transaction submitted: {}", ::substrate_primitives::hexdisplay::HexDisplay::from(&encoded));
```

Comment:
Probably add `target: "transaction-pool", ` (here and below) to conform with other traces in this module? Otherwise, iiuc, there'll be two differerent trace sources - `transaction-pool` and `polkadot_transaction_pool`.